README.md

```diff
@@ -19,7 +19,17 @@ Thank you, keep learing, keep building and happy hacktoberfest! :+1:
 
 ## Installation
 
-TDB...
+Clone the repo and install the dependencies.
+
+```
+git clone https://github.com/bradtaniguchi/es6-timed-map.git
+cd es6-timed-map
+```
+
+```
+nvm use
```

Comment:
This will error out for users if they don't have `nvm` installed, so not having it in the default instructions might be better. If one **does** have `nvm` installed, they probably know how to use `nvm use`, so having the file you added is still good. We can point out using `nvm` in a separate section under `## Installation`

Also I realized this is the in the [README.md](./README.md) rather than the [CONTRIBUTING.md ](./CONTRIBUTING.md)  which doesn't have an `Installation` section.

If you can move these changes to the CONTRIBUTING file under a new section `## Installation` that would be awesome :+1: